ci/build_scripts/build.sh

```diff
@@ -89,6 +91,14 @@ do
             BUILD=0
             ;;
 
+        --skip-test-packages)
+            INCLUDE_TEST_PACKAGES=0
+            ;;
+
+        --skip-deprecated-packages)
```

Comment:
```suggestion
        --include-deprecated-packages)
```
Only if we decide to exclude these packages by default, as suggested in the previous comment.